sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -2855,6 +2863,50 @@ private[sql] object QueryExecutionErrors extends QueryErrorsBase {
         "enumString" -> enumString))
   }
 
+  def unreleasedThreadError(loggingId: String, newAcquiredThreadInfo: String,
+                            AcquiredThreadInfo: String, timeWaitedMs: Long,
+                            stackTraceOutput: String): Throwable = {
```

Comment:
Fix indentations, see https://github.com/databricks/scala-style-guide#indent
```suggestion
  def unreleasedThreadError(
      loggingId: String,
      newAcquiredThreadInfo: String,
      AcquiredThreadInfo: String,
      timeWaitedMs: Long,
      stackTraceOutput: String): Throwable = {
```